e2e/integrations/edit/datamapper.po.ts

```diff
@@ -48,18 +48,13 @@ export class DataMapperComponent implements SyndesisComponent {
     return Number.parseInt(found[1]);
   }
 
-
   async createMapping(source: string, target: string): P<any> {
     log.info(`creating mapping from ${source} to ${target}`);
     const columns = await this.dataMapperColumns();
     const src = columns[0];
     const dest = columns[1];
 
-
-    return P.all([
-      this.selectMapping(source, src),
-      this.selectMapping(target, dest),
-    ]);
+    return P.all([this.selectMapping(source, src), this.selectMapping(target, dest)]);
```

Comment:
Personally I don't like the way code looks after this reformat :(
for instance here I wanted to have different function calls on different lines and there is no way around that reformat :(
```ts
    return P.all([		
      this.selectMapping(source, src),		
      this.selectMapping(target, dest),		
    ]);
```